### ISSUE TITLE
HGPS - Possible Format Error

### ISSUE THREAD

**Issue by tobiz** [Turbo87/aerofiles#18]
Should the HGPS record be, HGPS:MANUFACTURERSNAME,MODEL,CHANNELS,MAXALT ie have a ":" after HGPS?
This is the example shown in http://carrier.csi.cam.ac.uk/forsterlewis/soaring/igc_file_format/igc_format_2008.html#link_3.3 but the table below it says:

"_HFGPS | HFGPSMANUFACTURER,MODEL,CHANNELS,MAXALT | HFGPSMarconi,SuperX,12ch,10000m | 	Manufacturer and model of the GPS receiver used in the logger. I think there's a TYPO in this section of the FAI document (2008), suggesting a colon is needed._"

So there is a contradiction in the document and a potential error already noted. If there is an error and it should be HGPS: then line 301, "self.write_fr_header('GPS', gps_receiver)" should be:
"self.write_fr_header('GPS', gps_receiver, subtype_long="")"  or something to get the colon added.

Basically there is something not right around HGPS and it might be the spec. Ideally whatever is wrong, where ever it is needs to be corrected (I'm not saying your code is wrong, just in total something is not right).  If you believe your existing code is correct I'll contact the author of the spec and get them to change/clarify it.

**Comment by Turbo87** [comment]
the official spec can be found at http://www.fai.org/gnss-recording-devices/igc-approved-flight-recorders

if I understand the spec and the examples correctly the colon is only needed if the long form `HFGPSRECEIVER:foobar` is used, but not for `HFGPSfoobar`.

**Comment by tobiz** [comment]
Ok,

Thanks, understood.

Rgds

On 16 March 2016 11:27:16 GMT+00:00, Tobias Bieniek <notifications@github.com> wrote:
>the official spec can be found at
>http://www.fai.org/gnss-recording-devices/igc-approved-flight-recorders
>
>if I understand the spec and the examples correctly the colon is only
>needed if the long form `HFGPSRECEIVER:foobar` is used, but not for
>`HFGPSfoobar`.
>
>---
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/Turbo87/aerofiles/issues/18#issuecomment-197274175

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.
